fix: `updated_at` time stamp modification on updates.

Abiji-2020 opened the pull request:
## Summary

I have modified the column update in the database record field, whenever we update a new field along with the updated values, the `updated_at` timestamp will be updated and sent to the backend. 

This PR fixes #374 

## How did you test this change?

<!-- Describe how you tested this PR -->

I have tested this manually.


<img width="1044" height="156" alt="image" src="https://github.com/user-attachments/assets/b34c3797-9083-43b5-8565-a954175bb732" />
the updated_at timestamp's are automatically updated when i have modifed the nicknames, bio and avatar_urls.


<!-- This is an auto-generated comment: release notes by coderabbit.ai -->
## Summary by CodeRabbit

* **Chores**
  * Added a safe, idempotent database migration to ensure user "last updated" timestamps are maintained automatically, improving data consistency and reliability and providing clear deployment notices.
<!-- end of auto-generated comment: release notes by coderabbit.ai -->

Abiji-2020 commented:
Yeah, that makes sense. A DB-level trigger would handle updated_at more reliably.

Fermionic-Lyu commented:
> @Fermionic-Lyu I have created a migration file which adds the trigger to the users table which is created in the first migration script and removed manual timestamp updation from the frontend code.

Thanks for the change. However I think there is a pre-defined handy update trigger in migration file 000. You can check other migration files to understand how to use it. I would like to reuse that

Abiji-2020 commented:
> > @Fermionic-Lyu I have created a migration file which adds the trigger to the users table which is created in the first migration script and removed manual timestamp updation from the front-end code.
> 
> Thanks for the change. However I think there is a pre-defined handy update trigger in migration file 000. You can check other migration files to understand how to use it. I would like to reuse that

I’ve updated the migration to align with the existing migration format and conventions in the codebase.
Previously, I had modified it to use the trigger function but not in the standard migration flow.
With this commit, I’ve refactored it to correctly follow the migration structure and reuse the predefined trigger function from migration-000

Fermionic-Lyu reviewed (approved):
Changes LGTM!
@tonychang04 help make sure

tonychang04 commented:
can you comment on the original issue so we can assign it to you? This helps with our tracking purposes! Thanks!

Abiji-2020 commented:
> can you comment on the original issue so we can assign it to you? This helps with our tracking purposes! Thanks!

I have commented on this original issue.

tonychang04 commented:
tiny nit can align with the naming convertion if we use trigger we apply

update__
to 2 dashes?

else LGTM and i will merge soon, i promise :)

Abiji-2020 commented:
> tiny nit can align with the naming convertion if we use trigger we apply
> 
> update__ to 2 dashes?


On checking with the previous naming conventions we are having update__ 2 dashes because of the name of the table 
https://github.com/InsForge/InsForge/blob/ba684b5bf670c227d13e00a175dc907467d86d7e/backend/migrations/000_create-base-tables.sql#L125-L135

tonychang04 commented:
ahh goccha i stand corrected
